myecommerce/src/CartSite.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
```

Comment:
all these class components look great and ready to mapStateToProps and mapDispatchToProps and then connect. Awesome